browser/ui/views/frame/brave_tab_strip_region_view.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_FRAME_BRAVE_TAB_STRIP_REGION_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_FRAME_BRAVE_TAB_STRIP_REGION_VIEW_H_
+
+#include <memory>
+#include <string>
+#include "chrome/browser/ui/views/frame/tab_strip_region_view.h"
+
+class BraveTabStripRegionView : public TabStripRegionView {
```

Comment:
@nullhook I think we can archieve w/o overriding/patching.
`BrowserView` has `GetTabSearchButton()` method that can be used by our `BraveBrowserView`.
How about controlling prefs from `BraveBrowserView` ?

Reply:
It looks like `GetTabSearchButton()` is deleted from `Browser` from upstream.
Using `tab_strip_region_view()->tab_search_button()` from `BraveBrowserView` will work.